legislation.tex

```diff
@@ -86,5 +86,6 @@
         \item Loosing office key fob.
         \item Occupying a meeting room booked by another person (without leaving immediately).
         \item If network down > 15 min, or network down > 2 times on one day (calendar days, not 24 hours) POP2 will provide cake.
+        \item Incident with a duration longer than 8 hours or requiring action from Clearhaus employees on duty, fully or primarily caused by upstream service provider (\emph{cake responsible organization} is Clearhaus A/S. The preferred type of cake for this rule is icecream).
```

Comment:
This should be `requiring action from Clearhaus employees on duty outside of normal office hours`

Reply:
> normal office hours

That can be very hard to define and identify 😉

Reply:
Yes. That is why it's important with a review process for all important legislation. This is the 'høringsperiode' for this particular change.

Reply:
It's never only _on duty_ :sweat_smile: